src/client/netsblox-dev.html

```diff
@@ -25,6 +25,10 @@
         <script type="text/javascript" src="Snap--Build-Your-Own-Blocks/FileSaver.min.js"></script>
 
         <!-- Snap overrides -->
+        <script type="text/javascript" >
+            const SERVER_ADDRESS = 'https://editor.netsblox.org',
+                SERVER_HOST = SERVER_ADDRESS.replace(/^.*\/\//, '');
```

Comment:
Also, it might make more sense to call the url (with the protocol) `SERVER_URL` and use `SERVER_ADDRESS` for the address without the protocol (rather than `SERVER_HOST`)

Reply:
we could do that. I named them after location.host and location.origin